src/cmd/src/standalone.rs

```diff
@@ -416,4 +437,37 @@ mod tests {
         let toml_string = toml::to_string(&opts).unwrap();
         let _parsed: StandaloneOptions = toml::from_str(&toml_string).unwrap();
     }
+
+    #[test]
+    fn test_logging_options_toml() {
+        let cmd = StartCommand {
+            http_addr: None,
+            rpc_addr: None,
+            mysql_addr: None,
+            prom_addr: None,
+            postgres_addr: None,
+            opentsdb_addr: None,
+            config_file: Some(format!(
+                "{}/../../config/standalone.example.toml",
+                std::env::current_dir().unwrap().as_path().to_str().unwrap()
+            )),
+            influxdb_enable: false,
+            enable_memory_catalog: false,
+            tls_mode: None,
+            tls_cert_path: None,
+            tls_key_path: None,
+            user_provider: None,
+        };
+
+        let subcmd = SubCommand::Start(cmd);
+        let opt_res = subcmd.get_logging_options();
+        assert!(opt_res.is_ok());
+        let opt = opt_res.unwrap();
+        assert_eq!("debug", opt.logging.level);
+        assert_eq!("/tmp/greptimedb/logs", opt.logging.dir);
+        assert!(!opt.logging.enable_jaeger_tracing);
+        let _guard = logging::init_global_logging("standalone", &opt);
+        logging::debug!("test!");
+        logging::info!("{:?}", opt)
```

Comment:
suggestion: Remove them. We only need to assert the options are parsed correctly.
```suggestion
```